src/Automation/SnapshotCommand.cs

```diff
@@ -88,29 +73,11 @@ public static SnapshotCommandResult Execute(Config config, IOutputFileHelper out
                                 if (locationHelper.IsSarifExtension())
```

Comment:
Should have one more indent
```suggestion
                                    if (locationHelper.IsSarifExtension())
```